src/gpu/generic/sycl/sycl_primitive_conf.hpp

```diff
@@ -380,7 +380,8 @@ struct sycl_pooling_fwd_conf_t : public sycl_pooling_base_conf_t {
     sycl_post_ops_t post_ops;
 };
 
-#define DNNL_REF_SUM_MAX_NUM_TENSORS 16
+// Intel SYCL implementation supports a maximum of 8 tensors.
```

Comment:
I think it would be better to say that on Intel GPU the kernel fails to compile with more.

Reply:
Done.

Reply:
BTW, has this issue been raised to the sycl runtime? Do we know why this is specific to Intel GPU?

Reply:
We didn't raise the issue directly to SYCL runtime. Unfortunately, we don't know why this is happening to Intel GPU specifically.